src/components/data-management/AnalysisModal.jsx

```diff
@@ -45,6 +49,81 @@ const NewExperimentModal = (props) => {
     rules.ALPHANUM_DASH_SPACE,
   ];
 
+  const renderAnalysisList = () => {
+    if (!experimentsList?.length > 0) {
+      return (
+        <Row justify='center'>
+          <ClipLoader size={30} color='#8f0b10' />
+        </Row>
+      );
+    }
+
+    return (
+      <List
+        size='small'
+        bordered
+        dataSource={experimentsList}
+        itemLayout='vertical'
+        renderItem={(experiment) => (
+          <List.Item
+            key={`${experiment.id}`}
+            extra={(
+              <Row type='flex' align='middle' data-test-class='launch-analysis-item'>
+                <Col>
+                  <Button
+                    type='primary'
+                    onClick={() => {
+                      setIsWorking(true);
+                      onLaunch(experiment.id);
+                    }}
+                    disabled={numFieldsEditing > 0 || isWorking}
+                    onKeyDown={(e) => {
+                      if (e.key === 'Enter') {
+                        setIsWorking(true);
+                        onLaunch(experiment.id);
+                      }
+                    }}
+                  >
+                    Launch
+                  </Button>
+                </Col>
+              </Row>
+            )}
+          >
+            <Space direction='vertical' size='small'>
+              <strong>
+                <EditableField
+                  onAfterSubmit={async (name) => {
+                    dispatch(updateExperiment(experiment.id, { name: name.trim() }));
+                  }}
+                  value={experiment.name}
+                  validationFunc={(name) => validateInputs(name, validationChecks).isValid}
+                  deleteEnabled={false}
+                  onEditing={(editing) => {
+                    setNumFieldsEditing(Math.max(0, numFieldsEditing + (editing || -1)));
+                  }}
+                />
+              </strong>
+              <EditableField
+                onAfterSubmit={async (description) => {
+                  dispatch(
+                    updateExperiment(experiment.id, { description: description.trim() }),
+                  );
+                  await dispatch(saveExperiment(experiment.id));
```

Comment:
Why await this? If you remove, remember to remove `async` too

Reply:
I'm not sure - it was there before and I didn't add it. I'll check to see what the impact of removing `async/await` have.

Reply:
There doesn't seem to be any effect, so I'm removing it.